specs/player_spec.rb

```diff
@@ -0,0 +1,163 @@
+require 'minitest/autorun'
+require 'minitest/reporters'
+require 'minitest/skip_dsl'
+
+require_relative '../lib/player'
+
+# Get that nice colorized output
+Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
+
+describe 'Player' do
+
+  describe '#name' do
+
+    it 'returns the instance variable name for instance of Player' do
+      angie = Scrabble::Player.new("Angie")
+      angie.must_respond_to :name
+      angie.name.must_equal "Angie"
+    end
+
+  end # describe name
+
+  describe '#plays' do
+    it "returns an an empty Array if no words have been played" do
+      angie = Scrabble::Player.new("player_six")
+      result = angie.plays
+      result.must_be_kind_of Array
+      result.must_equal []
+    end
+
+    it "returns an Array of the words played by the player" do
+      robert = Scrabble::Player.new("robert")
+      robert.play("first")
+      robert.play("second")
+      result = robert.plays
+      result.must_be_kind_of Array
+      result.must_equal ["first", "second"]
+    end
+
+  end # describe plays
+
+  describe '#play(word)' do
+
+    it "returns 0 for empty strings and inappropriate characters" do
+      debbie = Scrabble::Player.new("debbie")
+      debbie.play('').must_equal 0
+      debbie.play('$%.982').must_equal 0
+    end
+
+    it "adds the input word to the plays Array" do
+      maria = Scrabble::Player.new("maria")
+      maria.play("academy")
+      result = maria.plays
+      result.must_equal ["academy"]
+    end
+
+    it "returns false if player has already won" do
+      edwin = Scrabble::Player.new("edwin")
+      edwin.play("qqqq") # 40
+      edwin.play("zzzz") # 40
+      edwin.play("qzqz") # 40
+      result = edwin.play("whatever")
+      result.must_equal false
+    end
+
+    it "returns the score of the word" do
+      mike = Scrabble::Player.new("mike")
+      result = mike.play("qqqq")
+      result.must_equal 40
+    end
+
+  end # describe play(word)
+
+  describe '#total_score' do
+
+    it "returns the sum of scores of played words" do
+      steph = Scrabble::Player.new("steph")
+      steph.play("qqqq") # 40
+      steph.play("zzzz") # 40
+      result = steph.total_score
+      result.must_equal 80
+    end
+
+    it "returns 0 if no words have been played" do
+      kyle = Scrabble::Player.new("kyle")
+      result = kyle.total_score
+      result.must_equal 0
+    end
+
+  end # describe total_score
+
+  describe '#won?' do
+
+    it "returns false if the player does not have over 100 points" do
```

Comment:
I would like to see more precision around the "edge" between winning and not winning. What happens if the player has:

- 99 points
- 100 points
- 101 points